Automattic/jetpack
# Update/migrate global notices to context

## Proposed changes:

* There was one instance left where the redux version of global notices was being used, this PR replaces that with the context version of notices
* Remove notices from redux

### Other information:


## Jetpack product discussion

P2: p1HpG7-rcF-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Sandbox the ephemeral or jurassic.ninja site
3. Purchase VideoPress for your site without connecting Jetpack to your user account
4. Go to My Jetpack and create an only site Jetpack connection (You can do this by following the connection flow until it asks you to Approve the user connection, at that point go back to My Jetpack)
![image](https://github.com/Automattic/jetpack/assets/65001528/1d358558-9b62-45f9-9731-6781a612b751)
5. When you get to My Jetpack, make sure the connection warning notice shows up
![image](https://github.com/Automattic/jetpack/assets/65001528/70c02acf-be45-4b53-a774-b71aee3f8a9f)
6. Next, in WPCOM, go to this class: uggcf%3N%2S%2Suers.yv%2S%3Suggcf%3N%2S%2Sbcratebx.n8p.pbz%2Sfbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qfgber%2Qncv%2Qraqcbvagf.cuc%3Se%3Q1prs6727%23780-og
and add return new WP_Error(); at the beginning of the callback function
7. Make sure the purchases error still shows up as it should
![image](https://github.com/Automattic/jetpack/assets/65001528/416c3b86-9b4e-4726-ae7a-a04b184ce6fe)
